nyxtools/flyer_eiger2.py

```diff
@@ -73,7 +76,7 @@ def configure_vector(self, **kwargs):
 
     def zebra_daq_prep(self):
         self.zebra.reset.put(1)
-        ttime.sleep(2.0)
+        ttime.sleep(2.0)  # TODO: very long sleep here
```

Comment:
this has been reduced in mxtools - look at https://github.com/NSLS-II-AMX/mxtools/commit/90e7755114dc0e7c31f9430011db0ed9438e4da0